dev/NumberBox/NumberBox.xaml

```diff
@@ -378,7 +472,20 @@
                                     </Storyboard>
                                 </VisualState>
                                 <VisualState x:Name="ButtonCollapsed" />
+                            </VisualStateGroup>
 
+                            <VisualStateGroup x:Name="SpinButtonStates">
```

Comment:
Should that be 'SpinButtonsStates' since you use 'SpinButtonsXXX' for the states?

Reply:
I think one plural is probably enough?